src/topics/registry-tree/networks.ts

```diff
@@ -16,4 +17,12 @@ export const networkChainIds: { [network in Network]: number } = {
   [Network.Gnosis]: 100,
   [Network.Polygon]: 137,
   [Network.Mumbai]: 80001,
+  [Network.ScrollTestnet]: 534353,
+};
+
+export const networkRpcUrls: { [network in Network]?: string } = {
```

Comment:
Don't understand why we need to defined them all whereas we only have the scroll that is using the wallet signer and we don't need alchemy. 
=> It's a bit strange to add alchemy env everywhere whereas it's not needed at all?